dipy/segment/bundles.py

```diff
@@ -67,9 +73,106 @@ def bundle_adjacency(dtracks0, dtracks1, threshold):
     return res
 
 
+def ba_analysis(recognized_bundle, expert_bundle, threshold=2.):
+    """ Calculates bundle adjacency score between two given bundles
+
+    Parameters
+    ----------
+    recognized_bundle : Streamlines
```

Comment:
Could you please add some more explanation of these? What's a recognized bundle? And what is an expert bundle? What is this a threshold for?